recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
 
 build:
   skip: true  # [python_impl == 'pypy']
```

Comment:
As [conda-forge dropped PyPy]( https://conda-forge.org/news/2024/08/14/sunsetting-pypy/ ), this is no longer needed

```suggestion
```